index.html

```diff
@@ -158,7 +167,8 @@ <h6 class="second_subtitle_card">2015</h6>
             </div>
 
             <div class="about_myself_section">
-                <h2 class="title">About Myself</h2>
+                <div class="about_me_1">
+                    <h2 class="title">About Myself</h2>
                 <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or
                     website.
                     Look through some of my work and experience! If you like what you see and have a project you need
```

Comment:
Please consider using the same styles for the paragraph as being used in the design like `letter spacing, font-size, etc` to make it look exactly like the design.

## Solution:-

Remember, to copy the styles of an element click on it in Figma and copy the styles from the right panel or side bar.